native-authentication/src/main/java/com/okta/nativeauthentication/OidcClientResultTransformer.kt

```diff
@@ -46,10 +46,12 @@ internal class OidcClientResultTransformer(
             is OidcClientResult.Success -> {
                 val response = result.result
                 if (response.isLoginSuccessful) {
-                    exchangeInteractionCodeForTokensAndEmit(response)
+                    coroutineScope.launch {
+                        exchangeInteractionCodeForTokensAndEmit(response)
```

Comment:
Why is this inside coroutineScope.launch now?

Reply:
For a polling action that results in a successful authentication. On success, we emit a loading event, and that cancels the previous jobs, effectively cancelling the token exchange. Now we launch the token exchange in the root scope (rather than the nested scope that's cancelled for polling).